deploy.go

```diff
@@ -232,14 +237,20 @@ func displayNewCommits(owner string, repo string, c *cli.Context, client *github
 	return nil
 }
 
-func newDeploymentRequest(c *cli.Context) (*github.DeploymentRequest, error) {
-	ref := Ref(c.String("ref"), git.Head)
+var EnvironmentAliases = map[string]string{
```

Comment:
I can make another pr which reads from `~/.deploy` (not sure about namespacing that one) or `DEPLOY_ENVIRONMENT_ALIASES`